sql/2022/jamstack/generate_queries.mjs

```diff
@@ -0,0 +1,52 @@
+// The queries in r1_jamstackishness_over_time are all very similar
+// This script generated them from a template
+// But the generated files are also checked in
```

Comment:
You can just either use the `almanac` data set (which by definition only has almanac years), or use this:"

```sql
SELECT
  IF(ENDS_WITH(_TABLE_SUFFIX, 'desktop'), 'desktop', 'mobile') AS client,
  REGEXP_REPLACE(_TABLE_SUFFIX, r'(\d+)_(\d+)_(\d+).*', r'\1-\2-\3') AS date,
  xxx
FROM
  `httparchive.requests.*
WHERE
  (_TABLE_SUFFIX like '2020_08_01%' OR _TABLE_SUFFIX like '2020_07_01%' OR _TABLE_SUFFIX like '2022_06_01%')
```

This only charges you for the tables you query.